Error in spl-dkms-git, makes zfs-dkms-git fail

After the dkms install of spl, this pops up:
```
cp: target '/var/lib/dkms/spl/2017.12.08.r3208.4e9b15696/4.14.0-rc5-jwrdegoede/x86_64/' is not a directory
```
(I use a patched kernel for the GPD Pocket, hence that version)

Afterwards, the zfs dkms install just fails:
```
configure: error:
    *** Please make sure the kmod spl devel package for your distribution
    *** is installed then try again.  If that fails you can specify the
    *** location of the spl source with the '--with-spl=PATH' option.
    *** The spl version must match the version of ZFS you are building,
    *** 0.7.0.  Failed to find spl.release.in in the following:
    300,
```

Funny thing is, this wasn't a problem with the old AUR packages, those installed fine.

--- a/src/kernels/dkms.sh
+++ b/src/kernels/dkms.sh
@@ -3,7 +3,7 @@ mode_name="dkms"
 mode_desc="Select and use the dkms packages"
 
 # version
-pkgrel="1"
+pkgrel="2"
 
 # Version for GIT packages
 pkgrel_git="${pkgrel}"
@@ -73,8 +73,8 @@ update_dkms_git_pkgbuilds() {
     fi
     spl_utils_pkgname="spl-utils-common-git=${spl_git_ver}"
     zfs_utils_pkgname="zfs-utils-common-git=${zfs_git_ver}"
-    spl_mod_ver=${spl_git_ver%%_*}
-    zfs_mod_ver=${zfs_git_ver%%_*}
+    spl_mod_ver="git"
+    zfs_mod_ver="git"
     spl_src_target="git+${spl_git_url}#commit=${latest_spl_git_commit}"
     zfs_src_target="git+${zfs_git_url}#commit=${latest_zfs_git_commit}"
 }

--- a/src/spl-dkms/PKGBUILD.sh
+++ b/src/spl-dkms/PKGBUILD.sh
@@ -21,9 +21,6 @@ ${spl_replaces}
 build() {
     cd "${spl_workdir}"
     ./autogen.sh
-    ./configure --prefix=/usr --libdir=/usr/lib --sbindir=/usr/bin \\
-                --with-config=user
-    make
 }
 
 package() {
@@ -32,8 +29,6 @@ package() {
     cp -a ${spl_workdir}/. \${dkmsdir}
 
     cd "\${dkmsdir}"
-    make clean
-    make distclean
     find . -name ".git*" -print0 | xargs -0 rm -fr --
     scripts/dkms.mkconf -v ${spl_mod_ver} -f dkms.conf -n spl
     chmod g-w,o-w -R .

--- a/src/spl-dkms/PKGBUILD.sh
+++ b/src/spl-dkms/PKGBUILD.sh
@@ -35,7 +35,7 @@ package() {
     make clean
     make distclean
     find . -name ".git*" -print0 | xargs -0 rm -fr --
-    scripts/dkms.mkconf -v ${zfs_mod_ver} -f dkms.conf -n spl
+    scripts/dkms.mkconf -v ${spl_mod_ver} -f dkms.conf -n spl
     chmod g-w,o-w -R .
 }
 

--- a/src/zfs-dkms/PKGBUILD.sh
+++ b/src/zfs-dkms/PKGBUILD.sh
@@ -22,10 +22,6 @@ ${zfs_replaces}
 build() {
     cd "${zfs_workdir}"
     ./autogen.sh
-    ./configure --prefix=/usr --sysconfdir=/etc --sbindir=/usr/bin --libdir=/usr/lib \\
-                --datadir=/usr/share --includedir=/usr/include --with-udevdir=/lib/udev \\
-                --libexecdir=/usr/lib/zfs-${zol_version} --with-config=user
-    make
 }
 
 package() {
@@ -34,8 +30,6 @@ package() {
     cp -a ${zfs_workdir}/. \${dkmsdir}
 
     cd "\${dkmsdir}"
-    make clean
-    make distclean
     find . -name ".git*" -print0 | xargs -0 rm -fr --
     scripts/dkms.mkconf -v ${zfs_mod_ver} -f dkms.conf -n zfs
     chmod g-w,o-w -R .

